example/README.md

```diff
@@ -15,10 +15,14 @@ Examples
 
 ---
 
-You can use dart_dev to serve the examples and run the required server with:
+To serve the examples:
 
+Dart 1:
 ```
-pub run dart_dev examples
+pub serve
 ```
 
-Then open [http://localhost:9000](http://localhost:9000) in Dartium or your browser of choice.
\ No newline at end of file
+Dart 2: TBD
+
```

Comment:
Can we just say `pub run build_runner serve example`  here ?

Reply:
Yes, but we will also need to upgrade to the version of over_react that eventually includes builder support. That's mainly why I was leaving this as TBD since the examples just won't work on Dart 2 right now